django/app_tutorial/models.py

```diff
@@ -52,47 +89,59 @@ def __str__(self):
                           team=self.team_id.name)
 
 
-class Tournament(models.Model):
-    name = models.CharField(max_length=75)
-    location = models.CharField(max_length=50)
-    date_begin = models.DateField('Begin Date')
-    date_end = models.DateField('End Date')
-    type = models.ForeignKey(TournamentType)
-    division = models.ForeignKey(Division)
+# TODO (Feedback): I don't think the way we wrote it down made much sense. Is this better?
```

Comment:
This looks like an advanced version of @mosespi's idea to normalize spirit scores -> :+1: